### PR TITLE
[DOC} Reconcile Alerting Monitors 2.9 Documentation Changes

### LINKED ISSUE #4708: [DOC] Resolve content and structure conflict issues
**What do you want to do?**
 
- [X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 - Address merge changes that were overridden
 - Address duplicative changes were published
 - Address section's information architecture (IA) updates that weren't published
 - Fix IA

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
#4525 
#4127

### PR DESCRIPTION
**Pull request by vagimeli**
… simultaneously

### Description
PRs #4525 and #4127 were worked on by two tech writers. Merged changes in 4525 were overridden but not incorporated by #4127. This PR is resolving that issue. This PR also resolves the Alerting Monitors info architecture as follows:

- [ ] Alerting (grand_parent)
- [ ] Monitors (parent)
- [ ] per document monitors (child)
- [ ] per cluster metrics monitors (child)
- [ ] per bucket monitors (child)
- [ ] per query monitors (child)
- [ ] composite monitors (child)
- [ ] triggers (child)
- [ ] actions (child)

### Issues Resolved
#4708 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@cwillum Destinations changed to Notifications. Deleted that section. Users go to Notifications documentation instead.

**Comment by vagimeli** [comment]
For visibility @mnkugler @brijos

File: _observing-your-data/alerting/monitors.md [hunk 1]
```diff
@@ -10,348 +10,24 @@ redirect_from:
...
 
-If your email provider requires SSL or TLS, you must authenticate each sender account before you can send an email. Enter these credentials in the OpenSearch keystore using the CLI. Run the following commands (in your OpenSearch directory) to enter your username and password. The `<sender_name>` is the name you entered for **Sender** earlier.
+1. **per query**: Runs a query and generates alert notifications based on the matching criteria. See [Per query monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/composite-monitors/) for information about creating and using this monitor type.
+1. **per bucket**: Runs a query that evaluates trigger criteria based on aggregated values in the dataset. See [Per bucket monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/composite-monitors/) for information about creating and using this monitor type.
+1. **per cluster metrics**: Runs API requests on the cluster to monitor its health. See [Per cluster metrics monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/composite-monitors/) for information about creating and using this monitor type.
```
Comment (cwillum) [inline _observing-your-data/alerting/monitors.md:21]:
```suggestion
1. **per cluster metrics**: Runs API requests on the cluster to monitor its health. See [Per cluster metrics monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/per-cluster-metrics-monitors/) for information about creating and using this monitor type.
```

File: _observing-your-data/alerting/monitors.md [hunk 2]
```diff
@@ -10,348 +10,24 @@ redirect_from:
...
+1. **per document**: Runs a query (or multiple queries combined by a tag) that returns individual documents that match the alert notification trigger condition. See [Per document monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/composite-monitors/) for information about creating and using this monitor type.
```
Comment (cwillum) [inline _observing-your-data/alerting/monitors.md:22]:
```suggestion
1. **per document**: Runs a query (or multiple queries combined by a tag) that returns individual documents that match the alert notification trigger condition. See [Per document monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/per-document-monitors/) for information about creating and using this monitor type.
```

File: _observing-your-data/alerting/monitors.md [hunk 3]
```diff
@@ -10,348 +10,24 @@ redirect_from:
...
-  <summary>
-    Table of contents
-  </summary>
-  {: .text-delta }
-- TOC
-{:toc}
-</details>
-
----
+Proactively monitor your data in OpenSearch with alerting and anomaly detection. Set up alerts to receive notifications when your data exceeds certain thresholds. Anomaly detection uses machine learning (ML) to automatically detect any outliers in your streaming data. You can pair anomaly detection with alerting to ensure that you're notified as soon as an anomaly is detected.
```
Comment (cwillum) [inline _observing-your-data/alerting/monitors.md:13]: If "alerting and anomaly detection" are in reference to the plugins (or, extension, now in the case of Anomaly Detection), I believe these mentions should be capitalized.

**Review by cwillum — COMMENTED** [review]
@vagimeli I looked over the monitors.md file and changed some links. Thanks.

File: _observing-your-data/alerting/per-cluster-metrics-monitors.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+- [_cat/indices]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-indices/)
+- [_cat/pending_tasks]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-pending-tasks/)
+- [_cat/recovery]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-recovery/)
+- [_cat/shards]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-shards/)
+- [_cat/snapshots]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-snapshots/)
+- [_cat/tasks]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-tasks/)
+
+## Restrict API fields
+
+If you want to hide fields from the API response that you do not want exposed for alerting, reconfigure the [supported_json_payloads.json](https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/resources/org/opensearch/alerting/settings/supported_json_payloads.json) file inside the Alerting plugin. The file functions as an allow list for the API fields you want to use in an alert. By default, all APIs and their parameters can be used for monitors and trigger conditions.
```
Comment (cwillum) [inline _observing-your-data/alerting/per-cluster-metrics-monitors.md:52]:
```suggestion
If you want to hide fields from the API response and not expose them for alerting, reconfigure the [supported_json_payloads.json](https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/resources/org/opensearch/alerting/settings/supported_json_payloads.json) file inside the Alerting plugin. The file functions as an allow list for the API fields you want to use in an alert. By default, all APIs and their parameters can be used for monitors and trigger conditions.
```

File: _observing-your-data/alerting/per-cluster-metrics-monitors.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  "indices": [
+    "shards.total",
+    "shards.index.shards.min"
+  ]
+}
+```
+
+## Painless triggers
+
+Painless scripts define triggers for cluster metrics monitors, similar to query or bucket-level monitors that are defined using the extraction query definition option. Painless scripts are comprised of at least one statement and any additional functions you wish to run.
```
Comment (cwillum) [inline _observing-your-data/alerting/per-cluster-metrics-monitors.md:67]:
```suggestion
Painless scripts define triggers for cluster metrics monitors, similar to per query or per bucket monitors, which are defined using the extraction query definition option. Painless scripts are composed of at least one statement and any additional functions you wish to run.
```

File: _observing-your-data/alerting/per-document-monitors.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- `is` 
+- `is not`
+- `is greater than`
+- `is greater than equal`
+- `is less than`
+- `is less than equal`
+
+You can query each [trigger]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/) using up to 10 tags, adding the tag as a single trigger condition instead of specifying a single query. The [Alerting plugin]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/) processes the trigger conditions from all queries as a logical `OR` operation, so if any of the query conditions are met, it triggers an alert. The Alerting plugin then tells the [Notifications plugin]({{site.url}}{{site.baseurl}}/observing-your-data/notifications/index/) to send the alert notification to a channel.
+
+You can only use _tags_, that is, labels that can be applied to multiple queries to combine them with the logical `OR`` operation, in a per document monitor.
```
Comment (cwillum) [inline _observing-your-data/alerting/per-document-monitors.md:33]:
```suggestion
You can only use _tags_--- that is, labels that can be applied to multiple queries to combine them with the logical `OR`` operation---in a per document monitor.
```
This feels like a parenthetical phrase. If the formatting looks wonky, I'd just throw them in actual parentheses.

File: _observing-your-data/alerting/per-query-bucket-monitors.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+title: Per query and per bucket monitors
+nav_order: 5
+parent: Monitors
+grand_parent: Alerting
+has_children: false
+---
+
+# Per query and per bucket monitors
+
+Per query monitors are a type of alert monitor that can be used to identify and alert on specific queries that are run against an OpenSearch index, for example, queries detect and respond to anomalies in specific queries. Per query monitors only trigger one alert at a time.
```
Comment (cwillum) [inline _observing-your-data/alerting/per-query-bucket-monitors.md:12]:
```suggestion
Per query monitors are a type of alert monitor that can be used to identify and alert on specific queries that are run against an OpenSearch index; for example, queries that detect and respond to anomalies in specific queries. Per query monitors only trigger one alert at a time. 
```

File: _observing-your-data/alerting/per-query-bucket-monitors.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+## Creating a per query or per bucket monitor
+
+To create a per query monitor, follow these steps:
+
+**Step 1.** Define your query and [triggers]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/). You can use any of these methods: visual editor, query editor, or anomaly detector.
+
+   - Visual definition works well for monitors that can be defined as "some value is above or below some threshold for some amount of time." It also works well for most monitors.
+
+   - Query definition provide flexibility in relation to your query (using [OpenSearch query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/index)) and how you evaluate the results of that query (Painless scripting).
```
Comment (cwillum) [inline _observing-your-data/alerting/per-query-bucket-monitors.md:24]:
```suggestion
   - Query definition provides flexibility in relation to your query (using [OpenSearch query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/index)) and how you evaluate the results of that query (Painless scripting).
```

File: _observing-your-data/alerting/per-query-bucket-monitors.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+           "boost": 1
+         }
+       },
+       "aggregations": {}
+     }
+```
+
+"Start" and "end" refer to the interval at which the monitor runs. See [Monitor variables]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/#monitor-variables).
+
+To define a monitor visually, choose **Visual editor**. Then choose a source index, a time frame, an aggregation (for example, `count()` or `average()`), a data filter (if you want to monitor a subset of your source index), and a group-by field if you want to include an aggregation field in your query. At least one group-by field is required if you're defining a bucket-level monitor.
```
Comment (cwillum) [inline _observing-your-data/alerting/per-query-bucket-monitors.md:73]:
```suggestion
To define a monitor visually, choose **Visual editor**. Then choose a source index, a time frame, an aggregation (for example, `count()` or `average()`), a data filter (if you want to monitor a subset of your source index), and a group-by field if you want to include an aggregation field in your query. At least one group-by field is required if you're defining a per bucket monitor. 
```

**Review by cwillum — APPROVED** [review]
Looks good. Suggestions included.

**Pull request merged**